mesheryctl/internal/cli/root/pattern/apply.go

```diff
@@ -309,6 +333,39 @@ func multiplePatternsConfirmation(profiles []models.MesheryPattern) int {
 		}
 	}
 }
+func getContextID(mctlCfg *config.MesheryCtlConfig) (string, error) {
```

Comment:
Since it is used in apply and delete sub-commands could you move implementation in pattern.go file

Reply:
sure

Reply:
Done